src/main/drivers/camera_control.c

```diff
@@ -108,10 +108,10 @@ void cameraControlInit()
             return;
         }
 
-        #ifdef USE_HAL_DRIVER
-        IOConfigGPIOAF(cameraControlRuntime.io, IOCFG_AF_PP, timerHardware->alternateFunction);
+        #ifdef STM32F1
```

Comment:
Camera control is not available on the F1

Reply:
The question is, does it work on F1? If so, then this should probably be left in, so that users can build custom F1 firmware with camera control by removing other features.